backend/utils/discourse/utils.js

```diff
@@ -17,27 +17,16 @@ function getMarkdown(project, proposal) {
     title: "# Project Category",
     body: categoryJson[project.projectCategory],
   });
-  console.log(
-    proposal.proposalEarmark,
-    earmarkJson,
-    earmarkJson[proposal.proposalEarmark]
-  );
   md.push({
     title: "# Proposal Earmark",
     body: earmarkJson[proposal.proposalEarmark],
   });
-  md.push({
-    title: "# Proposal Title",
-    body: proposal.oneLiner,
-  });
-
   if (proposal.proposalDescription)
     md.push({
-      title: "# Description",
+      title: "# Proposal Description",
```

Comment:
Updated this according to 
https://port.oceanprotocol.com/t/pr-pr-pr-project-pol-pol-pole-round-21/1778